site/content/3.12/develop/http-api/collections.md

```diff
@@ -950,6 +950,45 @@ logJsonResponse(response);
 db._drop(cn);
 ```
 
+### Get the available key generators
+
+```openapi
+paths:
+  /_api/key-generators:
+    get:
+      operationId: getKeyGenerators
+      description: |
+        Returns the available key generators for collections.
+
+        The call returns a JSON object with the available key generators
+        on success:
+
+        - `padded`
+        - `uuid`
+        - `autoincrement`
+        - `traditional`
+      responses:
+        '200':
+          description: |
+            The key generators that were found.
```

Comment:
I believe this is the correct way to describe the response. We should check what Swagger UI displays (should be a multi-select widget)

```suggestion
            The available generators for document keys.
          content:
            application/json:
              schema:
                type: array
                uniqueItems: true
                items:
                  type: string
                  enum:
                    - padded
                    - uuid
                    - autoincrement
                    - traditional
```